src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    String filePath;
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public ArrayList<Task> loadTasks() throws DukeException {
+        File f = new File(filePath);
```

Comment:
```suggestion
        File fileToLoad = new File(filePath);
```

Would it be better to specify a more descriptive variable name here? See '[Use standard words](https://nus-cs2103-ay2122s1.github.io/website/book/codeQuality/nameWell/basic/useStandardWords/)' & '[Use name to explain](https://nus-cs2103-ay2122s1.github.io/website/book/codeQuality/nameWell/intermediate/useNameExplain/)' in the textbook.